### DISCUSSION TITLE
open VB's quick icon at current tab

### DISCUSSION THREAD

**Discussion by bwisely** [k-ivan/visual-bookmarks-chrome#173] [General]
The searched result already supports opening with a new tab or current tab.
Can we support an option that opens VB's(visual bookmarks) quick icon in the current tab instead of opening a new tab?

Wisely

**Comment by k-ivan** [comment]
Hi!

Do you mean clicking the icon in the top right corner, as described #163 ?

**Reply by bwisely** [reply]
Yeah, I guess the issue is the same as my question.
When I click the icon, VB can support opening a new tab or using the current tab to open this click's web link. (That icon is a visual bookmark display for us to select which link to access quickly.)

**Reply by k-ivan** [reply]
Honestly, I didn't want to do this for two reasons:

1. I really don't like how it will have to be implemented in the code.
2. In the future, I might add additional functionality when clicking on the icon if I have any ideas. If that happens, the click on the icon will work completely differently.
